core/keyserver/keyserver.go

```diff
@@ -72,11 +72,15 @@ type WriteWatermark struct {
 
 // MutationLogs provides sets of time ordered message logs.
 type MutationLogs interface {
-	// Send submits an item to a random log.
+	// Send submits groups of item atomically to a random log.
+	// TODO(gbelvin): Create a batch level object to make it clear that this a batch of updates.
 	Send(ctx context.Context, directoryID string, mutation ...*pb.EntryUpdate) (*WriteWatermark, error)
-	// ReadLog returns the messages in the (low, high] range stored in the specified log.
+	// ReadLog returns the messages in the (low, high] range stored in the
+	// specified log.  ReadLog always returns complete units of the
```

Comment:
Please be clearer on what a "complete unit" is. Does this correspond to the batches submitted through `Send`?